docker-java-shaded/pom.xml

```diff
@@ -257,15 +257,6 @@
                             <include>com.google.guava:*</include>
                         </includes>
                     </artifactSet>
-                    <filters>
-                      <filter>
-                          <!-- JENKINS-39689: never embed tiger_types to not conflict with the one pulled by Stapler -->
-                          <artifact>*:*</artifact>
-                          <excludes>
-                            <exclude>org/jvnet/tiger_types/*</exclude>
-                          </excludes>
-                      </filter>
-                    </filters>
```

Comment:
How does this change relate to proposed improvement ? 
Wouldn't this resurect JENKINS-39689 ?